lib/testScenario/dataMasker.ts

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2021 Microsoft Corporation
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+import { inject, injectable } from "inversify";
+import { cloneDeep } from "@azure-tools/openapi-tools-common";
+import { TYPES } from "../inversifyUtils";
+import { setDefaultOpts } from "./../swagger/loader";
+
+type MaskFunction = (content?: string) => string;
+export interface DataMaskerOption {
+  maskValue: MaskFunction;
+}
+
+export const defaultMaskValue = (_content?: string): string => {
+  return "<masked>";
+};
+
+@injectable()
+export class DataMasker {
+  public maskKeys: string[] = [
```

Comment:
Should maskKeys be lower case strings?

Reply:
No. But macth function is case insensitive.